Makefile

```diff
@@ -14,20 +14,22 @@ $(SITE_DIR):
 	docker run \
 	-v $$(pwd)/src:/usr/src/app/src \
 	-v $$(pwd)/public:/usr/src/app/public \
+	-v $$(pwd)/index.html:/usr/src/app/index.html \
+	-v $$(pwd)/vite.config.mjs:/usr/src/app/vite.config.mjs \
 	-v $$(pwd)/package.json:/usr/src/app/package.json \
 	-v $$(pwd)/.env:/usr/src/app/.env \
 	-v $$(pwd)/$(SITE_DIR):/usr/src/app/$(SITE_DIR) \
 	-e CURRENT_UID=$(CURRENT_UID) \
-	-w /usr/src/app node:14-alpine \
-	sh -c 'npm install && REACT_APP_VERSION=$(VERSION) npm run build && chown -R $$CURRENT_UID: $(SITE_DIR)'
+	-w /usr/src/app node:18-alpine \
+	sh -c 'apk add --no-cache python3 make g++ && npm install && VITE_VERSION=$(VERSION) npm run build && chown -R $$CURRENT_UID: $(SITE_DIR)'
```

Comment:
Why `g++`?

Reply:
crypto libraries used require it: https://github.com/nodejs/node-gyp?tab=readme-ov-file#on-unix